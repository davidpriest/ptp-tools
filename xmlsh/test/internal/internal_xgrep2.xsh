# 
# Test of xgrep command

# XML output

xgrep -t / ../../samples/data/*.xml

