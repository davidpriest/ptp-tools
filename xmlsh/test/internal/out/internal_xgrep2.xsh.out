C:/Work/DEI/xmlsh/trunk/samples/data/REC-xml-20060816.xml
C:/Work/DEI/xmlsh/trunk/samples/data/books.xml
C:/Work/DEI/xmlsh/trunk/samples/data/intl.xml
C:/Work/DEI/xmlsh/trunk/samples/data/othello.xml
C:/Work/DEI/xmlsh/trunk/samples/data/xi_doc1.xml
C:/Work/DEI/xmlsh/trunk/samples/data/xi_sub1.xml
C:/Work/DEI/xmlsh/trunk/samples/data/xml-base-chap.xml
C:/Work/DEI/xmlsh/trunk/samples/data/xml-base-test.xml
